sktime/forecasting/hf_transformers_forecaster.py

```diff
@@ -105,6 +149,7 @@ def __init__(
         fit_strategy="minimal",
         validation_split=0.2,
         config=None,
+        peft_config=None,
```

Comment:
we need to add this as the past param, or we would have to go through deprecation.

Reply:
https://www.sktime.net/en/stable/developer_guide/deprecation.html
> adding more arguments at the end of the argument list, with a default value that retains prior behaviour, as long as the new arguments are well-documented

From my understanding, it does not require deprecation here, can you explain me if I'm wrong?

Reply:
it is not the last argument, so it would break positional calls.

Reply:
oh yes makes sense, it would break positional calls

Reply:
So do you suggest, I bring the first cycle in a separate PR and use this PR in the second cycle?

Reply:
I suggest to add it as the last argument, that way we avoid deprecation altogether